src/Emsi.Api/SkillEndpoint.cs

```diff
@@ -1,8 +1,13 @@
-﻿using Emsi.Playground.Dtos;
+using Emsi.Api.Dtos;
 using System.Threading.Tasks;
 
-namespace Emsi.Playground
+namespace Emsi.Api
 {
+    public class OccupationEndpoint
```

Comment:
Hi @VaclavElias  we can remove this class - OccupationEndpoint if it is not needed

Reply:
We will do clean up later, as another task, please approve :)